python/tvm/relay/op/transform.py

```diff
@@ -0,0 +1,23 @@
+"""Transform operators."""
+
+from . import _make
+
+
+def expand_dims(data, axis, num_newaxis=1):
+    """Insert `num_newaxis` axises at the position given by `axis`.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data to the operator.
+
+    axis : int
+        The axis at which the input array is expanded.
+        Should lie in range `[-data.ndim - 1, data.ndim]`.
+        If `axis < 0`, it is the first axis inserted;
+        If `axis >= 0`, it is the last axis inserted in Python's negative indexing.
+
+    num_newaxis : int
+        Number of axises to be inserted. Should be >= 0.
+    """
```

Comment:
```
Returns 
--------
```

Reply:
@tqchen Sorry for my poor English...What does this sentence mean: "The number of dimensions is num_newaxis greater than that of the input."?

Reply:
maybe result_dimension = source dimension  + num_newaxis

Reply:
@tqchen Good point! Like this?

```
result : relay.Expr
    The reshaped result. `result.ndim = data.ndim + num_newaxis`.
```